DataAccess/Repositories/AppointmentRepository.cs

```diff
@@ -17,28 +17,37 @@ public AppointmentRepository(ClinicContext clinicContext)
         public async Task CreateAsync(Appointment appointment)
         {
             await _clinicContext.AddAsync(appointment);
-            await _clinicContext.SaveChangesAsync();
         }
 
         public async Task DeleteAsync(Appointment appointment)
         {
             _clinicContext.Remove(appointment);
-            await _clinicContext.SaveChangesAsync();    
         }
 
         public async Task<IEnumerable<Appointment>> GetAsync()
         {
-            return await _clinicContext.Appointments.ToListAsync();
+            var appointment = await _clinicContext.Appointments
+                .Include(appointment => appointment.AppointmentProcedures)
```

Comment:
If you want use procedure property in future, call ` .ThenInclude(ap => ap.Procedure) ` after Include